CMakeLists.txt

```diff
@@ -695,8 +695,27 @@ if(WITH_SYSTEM_NPM)
   if(NOT NPM_EXECUTABLE)
     message(FATAL_ERROR "Can't find npm.")
   endif()
+  # build mgr dashboard frontent requires Angular v15 supports node.js versions: 14.20.x, 16.13.x and 18.10.x
+  # And More information Please See https://angular.io/guide/update-to-version-15#angular-v15-supports-nodejs-versions-1420x-1613x-and-1810x
+  execute_process(
+    COMMAND
+      "sh" "-c"
+      "node -v"
```

Comment:
please find `node` first, and just run the found executable, like
```cmake
find_program(NODEJS_EXECUTABLE
  NAMES node nodejs)
if(NODEJS_EXECUTABLE
  execute_process(COMMAND
    ${NODEJS_EXECUTABLE} --version
    OUTPUT_VARIABLE NodeJs_VERSION
    ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
  # parse the output
endif()
```

Reply:
if we use nodeenv to setup the Node.js runtime, there is no need to check the node versoin, right?

Reply:
> if we use nodeenv to setup the Node.js runtime, there is no need to check the node version, right?

Yeah, Because the command about `nodeenv --verbose -p --node=16.20.1` will specify the version number.